ext/rinku/autolink.c

```diff
@@ -27,22 +27,21 @@
 #define strncasecmp	_strnicmp
 #endif
 
+// using supported protocol ?
 bool
 autolink_issafe(const uint8_t *link, size_t link_len)
 {
 	static const size_t valid_uris_count = 5;
 	static const char *valid_uris[] = {
-		"/", "http://", "https://", "ftp://", "mailto:"
+		"http://", "https://", "ftp://", "mailto:"
 	};
 
 	size_t i;
 
 	for (i = 0; i < valid_uris_count; ++i) {
 		size_t len = strlen(valid_uris[i]);
 
-		if (link_len > len &&
-			strncasecmp((char *)link, valid_uris[i], len) == 0 &&
-			rinku_isalnum(link[len]))
```

Comment:
I'm hesitant to remove this in case `autolink_issafe` gets another consumer. As the overarching "is this safe to auto-link?" method, I'd rather leave this check in.